### PR TITLE
fix(native): add note about setting user for native release health user stats

### PR DESCRIPTION
**Pull request by kahest**
fixes https://github.com/getsentry/sentry-native/issues/911

### CODE REVIEW

**Review by lizokm — COMMENTED** [review]
This note isn't showing up in the preview for some reason...

**Comment by kahest** [comment]
> This note isn't showing up in the preview for some reason...

I see it here: https://sentry-docs-git-kahest-native-user-release-health-hint.sentry.dev/platforms/native/enriching-events/identify-user

Screenshot 2023-12-13 at 11 21 30

File: src/platform-includes/enriching-events/set-user/native.mdx [hunk 1]
```diff
@@ -6,3 +6,9 @@ sentry_value_set_by_key(user, "ip_address", sentry_value_new_string("{{auto}}"))
 sentry_value_set_by_key(user, "email", sentry_value_new_string("jane.doe@example.com"));
 sentry_set_user(user);
 ```
+
+<Note>
+
+To see user statistics in [Release Health](/platforms/native/configuration/releases/#release-health), set at least one of `id`, `username`, or `email`.
+
+</Note>
```
Comment (lizokm) [inline src/platform-includes/enriching-events/set-user/native.mdx:14]:
```suggestion
<Note>

To see user statistics in [Release Health](/platforms/native/configuration/releases/#release-health), set at least one of the following: `id`, `username`, or `email`.

</Note>
```

**Review by lizokm — APPROVED** [review]
LGTM!

**Pull request merged**